count_server/web/templates/layout/app.html.eex

```diff
@@ -30,6 +30,7 @@
       </main>
 
     </div> <!-- /container -->
+    <script src="//code.jquery.com/jquery-1.11.3.min.js"></script>
```

Comment:
Tried this, wasn't working on first attempts.  I'm going to go ahead and keep it for now and try adding it to package.json at some later point.